docker-compose.yml

```diff
@@ -19,7 +19,7 @@ services:
         command: 'true'
 
     pg:
-        image: postgres
+        image: postgres:10
```

Comment:
As discussed, if PostgreSQL usees semver this is fine, if not it'll need pinning to `10.x`